src/bun.js/node/types.zig

```diff
@@ -4935,7 +4934,7 @@ pub const Process = struct {
                 bun.String.static("bun"),
             );
         } else {
-            const exe_path = std.fs.selfExePathAlloc(allocator) catch null;
+            const exe_path = bun.selfExePath() catch null;
```

Comment:
Above we handle errors with a pretty error for this. Here we `null`. Is the error likely an out of memory thing where we would do the `bun.outOfMemory()`?

Reply:
below this line there's an optional check and provide a fallback without needing to throw an error.

Reply:
its pretty unlikely for this to error, the current executable would have to have the permissions changed after it started